compiler-rt/lib/interception/tests/interception_win_test.cpp

```diff
@@ -793,6 +793,38 @@ TEST(Interception, EmptyExportTable) {
   EXPECT_EQ(0U, FunPtr);
 }
 
+const struct InstructionSizeData_t {
```

Comment:
I think just `InstructionSizeData` would be more common in llvm style

Reply:
Done